code/modules/gear_presets/pmc.dm

```diff
@@ -634,3 +634,73 @@
 	name = "Weyland-Yutani PMC (Site Director) | HvH"
 	human_versus_human = TRUE
 	headset_type = /obj/item/device/radio/headset/distress/PMC/hvh/cct
+
+//*****************************************************************************************************/
+
+/datum/equipment_preset/pmc/synth
+		name = "Weyland-Yutani PMC (Support Synthetic)"
+		flags = EQUIPMENT_PRESET_EXTRA
+
+		languages = list(LANGUAGE_ENGLISH, LANGUAGE_RUSSIAN, LANGUAGE_JAPANESE, LANGUAGE_YAUTJA, LANGUAGE_XENOMORPH, LANGUAGE_WELTRAUMDEUTSCH, LANGUAGE_NEOSPANISH, LANGUAGE_CHINESE)
+
+		skills = /datum/skills/synthetic
+		idtype = /obj/item/card/id/data
+		assignment = JOB_PMC_SYNTH
+		rank = JOB_PMC_SYNTH
+		role_comm_title = "SYN"
+
+
+/datum/equipment_preset/pmc/synth/load_name(mob/living/carbon/human/H, var/randomise)
+		H.gender = pick(50;MALE,50;FEMALE)
+		var/datum/preferences/A = new()
+		A.randomize_appearance(H)
+		if(H.gender == MALE)
+				H.f_style = "5 O'clock Shadow"
+
+		H.change_real_name(H, H.gender == MALE ? pick(first_names_male_clf) : pick(first_names_female_clf))
```

Comment:
clf?

Reply:
PMCs are still behind with the naming list, their names are set/located in pmc.dm instead of names.dm
replaced clf with colonist instead for now

Reply:
scratch previous comment, PMCs now follow the same naming system that UPP and CLF are now. synth is adopting the pmc names now wooo